CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## Supported Release 3.0.0
+### Summary
+
+This is a backwards incompatible major release.
+
+### Features
+
+- Docs updated for puppet-strings
+- Adds Puppet 4 data types and deprecation warnings
```

Comment:
historically we've only had `### Features` and `### Bugfixes` sections. I'm cool to change but we should make sure it happens across all modules. 

Also, I'm looking at the diff from 2.2.1 which is pretty slim. Should I be diff-ing from the release before that?